Aptoide/aptoide-client-v8
# [ASV-1396] Integrate start app

**What does this PR do?**

   This PR aims at implementing startapp network. Since it is not one of the certified networks (custom network) it had to be adapted for mopub 5.5, which means that we had to create a BaseAdapterConfiguration for StartApp and add it in the initialization of MoPub.
Also, due to apk size constraints we had to remove AdColony (previously we also removed tapjoy).

PS: StartApp ads are only for interstitials in AppView and Banners (home, search and appview)
**Database changed?**

    No

**Where should the reviewer start?**

- [ ] StartAppBaseAdapterConfiguration.java

**How should this be manually tested?**

Open the app and check if you see ads. If you want to do further testing, lock all the networks on mopub dashboard, leaving only startapp as active network.
Don't forget that you will need the key.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1396](https://aptoide.atlassian.net/browse/ASV-1396)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 

MOPUB_STARTAPP_APPLICATION_ID


**Code Review Checklist**
